src/tejashree/CompareTwoArrays.java

```diff
@@ -0,0 +1,29 @@
+/*Program 1: WAP to Find Common Element Between Two Arrays
+
+input: 
+
+int array1 [] = [1, 2, 5, 5, 8, 9, 7, 10]
+int array2 [] = [1, 0, 6, 15, 6, 4, 7, 0]
+
+output: 1 7
+*/
+
+package tejashree;
+
+public class CompareTwoArrays {
+
+	public void compareArrays(int []a1, int []a2) {
+		for(int index=0;index<a1.length;index++) {
```

Comment:
Hi Tejashree,
you need to compare element in two loop, fortunately both array have same common value on same index, but think on diff index,
Pls update accordingly.